src/GUTI/GUTIChecker.java

```diff
@@ -58,4 +57,10 @@ public InferenceAnnotatedTypeFactory createInferenceATF(
                 slotManager, constraintManager);
     }
 
+    @Override
+    public boolean isInsertMainModOfLocalVar() {
+        return true;
```

Comment:
I think we don't want this - local variables should be top and then refined automatically.